railties/lib/rails/generators/rails/scaffold/scaffold_generator.rb

```diff
@@ -16,13 +16,10 @@ class ScaffoldGenerator < ResourceGenerator # :nodoc:
       def handle_skip
         @options = @options.merge(stylesheets: false) unless options[:assets]
         @options = @options.merge(stylesheet_engine: false) unless options[:stylesheets] && options[:scaffold_stylesheet]
-        @options = @options.merge(system_tests: false) if options[:api]
```

Comment:
I swapped this out for `@options = @options.merge(system_tests: nil) if options[:api]` and it still wasn't available in the options hash in railties/lib/rails/generators/test_unit/scaffold/scaffold_generator.rb so I left the `options.api?` check in. I'm not sure what the purpose of this line is.